ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/parts/EditorTab.java

```diff
@@ -46,6 +46,14 @@
     void setFile(VirtualFile file);
 
     /**
+     * Set color to the label of editor's tab.
+     *
+     * @param color
+     *         color to set
+     */
+    void setTitleColor(String color);
```

Comment:
Do you think ti makes sense to specify the format of the color in the javadoc, like _hash #000000_ or color name like _blue_ ?